.github/workflows/ci.yml

```diff
@@ -82,6 +83,27 @@ jobs:
         run: |
           python -c "import ansys.tools.report as report; report.version();"
 
+      - name: Retrieve version
+        run: |
+          echo "::set-output name=PYANSYS_TOOLS_REPORT_VERSION::$(python -c "import ansys.tools.report as report; print(report.version())")"
+        id: version
```

Comment:
I like this!

Reply:
Sure you like it... it's coming from PyMAPDL 😄 thanks guys for finding that out!